documentation/modules/cruise-control/con-optimization-goals.adoc

```diff
@@ -30,66 +30,91 @@ For more information on each optimization goal, see link:https://github.com/link
 
 NOTE: CPU goals, intra-broker disk goals, "Write your own" goals, and Kafka assigner goals are not yet supported.
 
-As described in the following sections, you can customize the supported optimization goals by reordering them in terms of priority, and disabling goals to exclude from optimization proposal calculations.
-
 [discrete]
-== Goals configuration in {ProductName}
+== Goals configuration in {ProductName} custom resources
 
-You configure optimization goals in the `Kafka` and `KafkaRebalance` custom resources. Cruise Control has configurations for link:#hard-soft-goals[hard] optimization goals that must be satisfied, as well as link:#master-goals[master], link:#default-goals[default], and link:#user-provided-goals[user-provided] optimization goals.
+You configure optimization goals in `Kafka` and `KafkaRebalance` custom resources. Cruise Control has configurations for xref:#hard-soft-goals[_hard_] optimization goals that must be satisfied, as well as xref:#master-goals[_master_], xref:#default-goals[_default_], and xref:#user-provided-goals[_user-provided_] optimization goals. 
+Optimization goals are subject to any xref:#capacity-configuration[_capacity limits_] on broker resources.
 
 The following sections describe each goal configuration in more detail.
 
 [#hard-soft-goals]
 [discrete]
 === Hard goals and soft goals
 
-_Hard goals_ are goals that _must_ be satisfied in optimization proposals. 
-Goals that are not hard goals are known as _soft goals_, and might not be satisfied by optimization proposals. 
+Hard goals are goals that _must_ be satisfied in optimization proposals. 
+Goals that are not configured as hard goals are known as _soft goals_. 
 You can think of soft goals as _best effort_ goals: they do _not_ need to be satisfied in optimization proposals, but are included in optimization calculations.
 An optimization proposal that violates one or more soft goals, but satisfies all hard goals, is valid.
 
-Cruise control will calculate optimization proposals that satisfy all the hard goals and as many soft goals as possible (in their priority order). 
+Cruise Control will calculate optimization proposals that satisfy all the hard goals and as many soft goals as possible (in their priority order). 
 An optimization proposal that does _not_ satisfy all the hard goals is rejected by Cruise Control and not sent to the user for approval.
 
 NOTE: For example, you might have a soft goal to distribute a topic's replicas evenly across the cluster (the topic replica distribution goal). 
 Cruise Control will ignore this goal if doing so enables all the configured hard goals to be met.
 
-In Cruise Control, six of the link:#master-goals[master optimization goals] are preset as hard goals:
+In Cruise Control, six of the xref:#master-goals[master optimization goals] are preset as hard goals:
```

Comment:
do we need to use a specific number here (six)? If something changes in the future with less or more hard goals I think we will totally forget to update this number. I am pretty sure :-) Maybe just using "some" ... or your call ;-)

Reply:
Will change to say "the following master optimization goals".

Also, there are five not six. D'oh!